src/src/rx_main.cpp

```diff
@@ -2219,7 +2219,7 @@ void loop()
         DBGLN("Req air rate change %u->%u", ExpressLRS_currAirRate_Modparams->index, ExpressLRS_nextAirRateIndex);
         if (!isSupportedRFRate(ExpressLRS_nextAirRateIndex))
         {
-            DBGLN("Mode %u not supported, ignoring", get_elrs_airRateConfig(index)->interval);
+            DBGLN("Mode %u not supported, ignoring", get_elrs_airRateConfig(ExpressLRS_currAirRate_Modparams->index)->interval);
```

Comment:
Personally I would change it to `DBGLN("Mode %u not supported, ignoring", ExpressLRS_nextAirRateIndex);`

Reply:
makes sense